src/AppInstallerCLICore/Workflows/DependenciesFlow.h

```diff
@@ -39,19 +39,24 @@ namespace AppInstaller::CLI::Workflow
     // Outputs: Dependencies
     void GetDependenciesInfoForUninstall(Execution::Context& context);
 
-    // Builds the dependency graph.
+    // Builds the dependency graph and creates the sub contexts for each package dependency.
     // Required Args: None
     // Inputs: Manifest, Installer and DependencySource
     // Outputs: Dependencies
-    struct ManagePackageDependencies : public WorkflowTask
+    struct CreateDependencySubContexts : public WorkflowTask
     {
-        ManagePackageDependencies(AppInstaller::StringResource::StringId dependencyReportMessage) :
-            WorkflowTask("ReportDependencies"), m_dependencyReportMessage(dependencyReportMessage) {}
+        CreateDependencySubContexts(
+            AppInstaller::StringResource::StringId dependencyReportMessage,
+            bool includeInstalledPackages = false) :
+            WorkflowTask("ReportDependencies"),
+            m_dependencyReportMessage(dependencyReportMessage),
+            m_includeInstalledPackages(includeInstalledPackages) {}
 
         void operator()(Execution::Context& context) const override;
 
         private:
             AppInstaller::StringResource::StringId m_dependencyReportMessage;
+            bool m_includeInstalledPackages;

```

Comment:
Now I just realized if we create the source without the installed source, this bool may not be needed at all.